src/Nethermind/Nethermind.Serialization.Json/EthereumJsonSerializer.cs

```diff
@@ -16,21 +16,20 @@ namespace Nethermind.Serialization.Json
 {
     public class EthereumJsonSerializer : IJsonSerializer
     {
+        public const int DefaultMaxDepth = 128;
```

Comment:
In https://github.com/NethermindEth/nethermind/blob/f2a86cc50c86194d3044256d473068923e491506/src/Nethermind/Nethermind.JsonRpc/JsonRpcConfig.cs#L67 we are using the `EthereumJsonSerializer.DefaultMaxDepth` constant. I don't think it will change any time soon but it would be nice if we used the same source for the constant throughout the system to avoid potential mismatches.

Reply:
Eee this is the same constant!

Reply:
My bad, I got confused with the source of the constant.